.github/workflows/ci.yml

```diff
@@ -290,11 +290,11 @@ jobs:
        export LD_LIBRARY_PATH=$PWD/build/bin/opencv_libs:/usr/lib/x86_64-linux-gnu
        echo "LD_LIBRARY_PATH: $LD_LIBRARY_PATH"
     
-       LD_DEBUG=files ./build/bin/ACC_MNIST* 2> ld_debug.log
+       LD_DEBUG=files ./build/bin/ACC* --model alexnet_mnist 2> ld_debug.log
        echo "### Library loading debug ###"
        grep -i "opencv_imgcodecs" ld_debug.log
 
-       ./build/bin/ACC_MNIST* > accuracy.txt
+       ./build/bin/ACC* > accuracy.txt
```

Comment:
Avoid `./build/bin/ACC*` wildcard for accuracy run; use the explicit binary path